identity-apps-core/apps/authentication-portal/src/main/webapp/requested-claims.jsp

```diff
@@ -258,9 +457,22 @@
 
     $(document).ready(function () {
 
-        $("input[name^='claim_mand_']").on("input", function () {
+        $("input[name^='claim_mand_']").on("input change", function () {
             var claimId = $(this).attr("name");
-            if ($(this).val()) {
+            var inputType = $(this).attr("type");
+
+            if (inputType === "checkbox") {
```

Comment:
Shall we remove this logic since there is no validation for the checkbox fields?